### PR TITLE
[usecase-linux][xwalk-3716] Add usecase for display mode

### PR DESCRIPTION
**Pull request by kangxu**
- Verify the display mode support fullscreen, standalone,
minimal-ui(default)

Impacted tests(approved): new 1, update 0, block 0
Unit test platform: [Linux]
Unit test result summary: pass 1, fail 0, block 0

### CODE REVIEW

File: usecase/usecase-wrt-linux-tests/res/testapp/org.display.defaultmode/manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "display_standalone",
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/org.display.defaultmode/manifest.json]: s/standalone/default/

File: usecase/usecase-wrt-linux-tests/samples/DisplayMode/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Usecase Design
+
+This sample demonstrates the support for setting csp policies in manifest.json.
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/DisplayMode/README.md]: s/the support for setting csp policies/display mode support/

Add a line for the spec: http://w3c.github.io/manifest/#display-member

File: usecase/usecase-wrt-linux-tests/samples/DisplayMode/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <h2>Test Steps:</h2>
+      <ol>
+        <li>Install the webapp 'fullscreen.deb' and launch it</li>
+        <li>Install the webapp 'minimalui.deb' and launch it</li>
+        <li>Install the webapp 'standalone.deb' and launch it</li>
+        <li>Install the webapp 'defaultmode.deb' and launch it</li>
+      </ol>
+      <h2>Expected Result:</h2>
+      <ol>
+        <li>Test app should be displayed in fullscreen mode</li>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/DisplayMode/index.html]: s/should be displayed/is launched/g

File: usecase/usecase-wrt-linux-tests/samples/DisplayMode/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        <li>Test app should be displayed in minimal-ui mode</li>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/DisplayMode/index.html]: Please explain what is <fullscreen|minimal-ui/standalone> mode. End tester doesn't care about the display mode. They just care about what is shown and what to check.

**Comment by kangxu** [comment]
@zqzhang 
I have updated it following your comments

**Pull request merged**